packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -93,13 +93,26 @@ and GCed, the victims it referenced will be also GCed:
 
 ### Limitations
 
-The tool detects leaks for disposable and instrumented classes only.
-(Note that fixing the leak can also fix other objects.)
+The tool detects leaks for disposable and instrumented classes only
+(note that a fix for those detected leaks can also fix some not detected
+leaks).
 
-Some classes in the Flutter framework are already instrumented.
-If you want your classes to be tracked, you need to make them
+Some classes in the Flutter framework and Dart SDK are already instrumented.
+If your application is leaking these classes, the leaks will be
+detected without additional effort.
+
+If you want your classes to be leak tracked, you need to make them
 disposable and [instrument](#instrument) them.
 
+### Performance impact
+
+The Leak Tracker stores a small additional record for each tracked alive object and for each
+detected leak, that increases the memory footprint.
+
+For the [Gallery application](https://github.com/flutter/gallery) in profile mode on `macos`
+the leak tracking increased memory footprint of the home page by ~400 KB that is ~0.5% of
+the total.
```

Comment:
Is there a noticeable impact on runtime performance?

Reply:
added